src/gui/gtk.c

```diff
@@ -1670,40 +1669,18 @@ void dt_ui_notify_user()
 
 static void _ui_init_panel_size(GtkWidget *widget)
 {
-  gchar *key = NULL;
-
-  int s = 128;
-  if(strcmp(gtk_widget_get_name(widget), "right") == 0)
-  {
-    key = _panels_get_panel_path(DT_UI_PANEL_RIGHT, "_size");
-    s = DT_UI_PANEL_SIDE_DEFAULT_SIZE; // default panel size
-    if(key && dt_conf_key_exists(key))
-      gtk_widget_set_size_request(widget, dt_conf_get_int(key), -1);
-  }
-  else if(strcmp(gtk_widget_get_name(widget), "left") == 0)
-  {
-    key = _panels_get_panel_path(DT_UI_PANEL_LEFT, "_size");
-    s = DT_UI_PANEL_SIDE_DEFAULT_SIZE; // default panel size
-    if(key && dt_conf_key_exists(key))
-      gtk_widget_set_size_request(widget, dt_conf_get_int(key), -1);
-  }
-  else if(strcmp(gtk_widget_get_name(widget), "bottom") == 0)
-  {
-    key = _panels_get_panel_path(DT_UI_PANEL_BOTTOM, "_size");
-    s = DT_UI_PANEL_BOTTOM_DEFAULT_SIZE; // default panel size
-    if(key && dt_conf_key_exists(key))
-      s = CLAMP(dt_conf_get_int(key), dt_conf_get_int("min_panel_height"), dt_conf_get_int("max_panel_height"));
-    if(key) gtk_widget_set_size_request(widget, -1, s);
-  }
+  gchar *key = _panels_get_panel_path(DT_UI_PANEL_BOTTOM, "_size");
+  int s = DT_UI_PANEL_BOTTOM_DEFAULT_SIZE; // default panel size
+  if(key && dt_conf_key_exists(key))
+    s = CLAMP(dt_conf_get_int(key), dt_conf_get_int("min_panel_height"), dt_conf_get_int("max_panel_height"));
+  if(key) gtk_widget_set_size_request(widget, -1, s);
```

Comment:
I had removed the if(key) as I don't see why it is needed at all. `s` is always initialized anyway.